docs/rules/dynamic-import-chunkname.md

```diff
@@ -0,0 +1,66 @@
+# dynamic imports require a leading comment with a webpackChunkName (dynamic-import-chunkname)
+
+This rule reports any dynamic imports without a webpackChunkName specified in a leading block comment in the proper format.
+
+This is a useful rule because if the webpackChunkName is not defined in a dynamic import, Webpack will autogenerate the chunk name.
+
+## Rule Details
+This rule runs against `import` by default, but can be configured to also run against an alternative dynamic-import function, e.g. 'dynamicImport.'
+You can also configure the regex format you'd like to accept for the webpackChunkName - for example, we don't want the number 6 to show up in our chunk names.
+ ```javascript
+{
+  "dynamic-import-chunkname": [2, {
+    importFunction: "dynamicImport",
```

Comment:
I wonder if this should be `importFunctions` and take an array instead of a string. What do you think?